src/pages/info.js

```diff
@@ -0,0 +1,149 @@
+    content:
+      <div>
+        <p>You will get two vouchers for the <a href="#lunch">lunch</a> at the TU Mensa which include a starter, main + 2 side dishes, dessert and a non-alcoholic beverage. <emph>(voucher is not available for ticket level: "Free attendance for TU Berlin students")</emph></p>
+        <p>You will get an additional voucher for the <a href="#dinner">dinner</a> at BRLO Brwhouse which includes the a welcome drink + snack and a 4-course meal. <emph>(voucher is not available for ticket levels: "Day ticket, Only conference (late bird), Press, Free attendance for TU Berlin students")</emph></p>
+      </div>
+  },
+  {
+    id: "lunch",
+    title: "Lunch",
+    content:
+      <p>We will have lunch together on the first two days at the <ExternalLink href="https://goo.gl/maps/PK8zi41DgrK2">TU Mensa</ExternalLink> from roughly 12:30 each day. At the end of the first block of talks we will walk there together. Make sure to bring your voucher! There is no provided lunch on the third day of the conference.</p>
+  },
+  {
+    id: "concert",
+    title: "Concert",
+    content:
+      <div>
+        <p>
+          The concert is taking place in the basement of <ExternalLink href="https://www.google.com/maps/place/52%C2%B032'11.9%22N+13%C2%B023'42.2%22E/@52.536626,13.3945158,19z/data=!3m1!4b1!4m6!3m5!1s0x0:0x0!7e2!8m2!3d52.5366255!4d13.395063">Factory Berlin</ExternalLink> in the SoundCloud HQ. The closest station is <strong>"U Bernauer Str"</strong> (serviced by U8 and M10). From there, walk south on <strong>Brunnenstr.</strong> until you can turn right into <strong>Rheinsberger Str.</strong>. Follow that street until you see WAC signs. It's a side entry of the Factory that leads to their event basement.
+        </p>
+        <p>
+          Doors open at 19:30 and the last admission is 20:00.
+        </p>
+        <emph>
+          Please also make sure that your phone is fully charged as some of the performances will include particapatory elements.
+        </emph>
+        <InfoImage
+          src={withPrefix('/info/directions-bernauer.jpg')}
+          alt="Directions from U Bernauer Str to the concert"
+        />
+      </div>
+  },
+  {
+    id: "dinner",
+    title: "Dinner",
+    content:
+      <div>
+        <p>
+          The dinner this year is at <ExternalLink href="https://goo.gl/maps/DcaC59cZbnu">BRLO Brwhouse</ExternalLink> which is located right next to <strong>"U Gleisdreieck"</strong> (serviced by U1, U2 and U3). From the station, exit towards <strong>Gleisdreieck Park</strong> and you will find BRLO as the first building right after the entrance at your left hand side.
+        </p>
+        <p>
+          Doors open from 19:00 with a welcome drink + snacks. Please don't arrive later than 19:30.
+        </p>
+        <emph>
+          Please make sure to bring your name tag and dinner voucher that you get at the registration and check that your ticket includes the dinner.
+        </emph>
```

Comment:
The two <emph> tags are rendered without a space in between them. Could you maybe add an extra &nbsp;?

Reply:
Ups the "& n b s p ;" gets swallowed by GitHub,

Reply:
nice catch, fixed it :)